pyproject.toml

```diff
@@ -5,10 +5,10 @@ build-backend = "setuptools.build_meta"
 [project]
 name = "niftyone"
 description = "Large-scale neuroimaging visualization using FiftyOne"
-authors = [{name = "Connor Lane", email = "connor.lane858@gmail.com"}]
+authors = [{ name = "Connor Lane", email = "connor.lane858@gmail.com" }]
```

Comment:
What was the cause of these nonsignificant diffs? Some version update of ruff or something?

Reply:
Must have been - I honestly didn't even notice this so it must have been formatted by the pre-commit.

Reply:
Yeah this is the TOML formatter in our pre-commit